NEWS.md

```diff
@@ -1,3 +1,25 @@
+### v10.3.2 (2023-06-26)
+
+#### Bug Fixes
+
+* **prisma:** check for query args in string property ([#1684](https://github.com/newrelic/node-newrelic/pull/1684)) ([8d9b176](https://github.com/newrelic/node-newrelic/commit/8d9b176e05424d34dd2eafe243a8cee7284efe64))
+
+#### Security Improvements
+
+* **deps:** bump fast-xml-parser and @aws-sdk/client-lambda ([#1697](https://github.com/newrelic/node-newrelic/pull/1697)) ([4376584](https://github.com/newrelic/node-newrelic/commit/437658415a2869ed4236fa31b57128bb9807a2f3))
+    * Signed-off-by: dependabot[bot] <support@github.com> Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
+
+#### Miscellaneous Chores
+
+* **deps:** Updated security agent to v0.1.3 ([#1696](https://github.com/newrelic/node-newrelic/pull/1696)) ([0f957bd](https://github.com/newrelic/node-newrelic/commit/0f957bd0e39a32614337b5ade33f07d9729f1897))
```

Comment:
also this was a security update right? If so, make sure you also edit the changelog.json to move this into the security section